.trunk/trunk.yaml

```diff
@@ -0,0 +1,59 @@
+# This file controls the behavior of Trunk: https://docs.trunk.io/cli
+# To learn more about the format of this file, see https://docs.trunk.io/reference/trunk-yaml
+version: 0.1
+cli:
+    version: 1.18.1
+# Trunk provides extensibility via plugins. (https://docs.trunk.io/plugins)
+plugins:
+    sources:
+        - id: trunk
+          ref: v1.4.0
+          uri: https://github.com/trunk-io/plugins
+# Many linters and tools depend on runtimes - configure them here. (https://docs.trunk.io/runtimes)
+runtimes:
+    enabled:
+        - go@1.21.0
+        - node@18.12.1
+        - python@3.10.8
+tools:
+    runtimes:
+        - node
+# This is the section where you manage your linters. (https://docs.trunk.io/check/configuration)
+lint:
+    definitions:
+        - name: eslint
+          commands:
+              - name: lint
+                run: ${workspace}/bin/eslint.js --output-file ${tmpfile} --format json ${target}
+    enabled:
+        - eslint@8.56.0
```

Comment:
I'm curious if this is needed given the previous definition?

Reply:
Yeah - the definition teaches trunk how to run a tool - and the enabled list turns on the tools.